### PR TITLE
Bug904139 - Add grab and grabbing icons for the timeline layers

### PR DESCRIPTION
**Pull request by maronin**
Tested in Chrome, Firefox, and IE10. Works in all three. 

Note: IE does not support the grab cursor like Chrome and Firefox, so I used custom icons I made myself when the user is on IE.

### CODE REVIEW

**Comment by mjschranz** [comment]
Hey Mark! Thanks for the pull request but the repository we use for development with popcorn.webmaker.org is at mozilla/popcorn.webmaker.org. This repository is closed/deprecated by and large.

Could you please reopen this there? Thanks!

**Comment by maronin** [comment]
Alright will do! :)

File: public/css/tray-timeline.less [hunk 1]
```diff
@@ -281,7 +281,9 @@
 .track-handle {
   border: none;
   border-bottom: 1px solid darken( @baseLight, 25% );
-  cursor: pointer;
+  cursor: url(/media/img/hand.cur), default; /*EI Support*/
```
Comment (ScottDowne) [inline public/css/tray-timeline.less:284]: I don't think I care enough about cursors in IE to add a custom cursor file.

Comment (maronin) [inline public/css/tray-timeline.less:284]: Ok, how about using google's grab cursor? using something like:

`cursor: url(https://mail.google.com/mail/images/2/openhand.cur)`
and
`cursor: url(https://mail.google.com/mail/images/2/closedhand.cur)`

I don't know about the licensing and if you can use those icons, so I'm not sure. 

Maybe IE can just have a `cursor: pointer` and a `cursor: move` instead?

File: public/css/tray-timeline.less [hunk 2]
```diff
@@ -281,7 +281,9 @@
...
+  cursor: -webkit-grab;                      /*Chrome Support*/
+  cursor: -moz-grab;                         /*Mozzila Support*/
```
Comment (ScottDowne) [inline public/css/tray-timeline.less:286]: Come FF 27, grab and grabbing are no longer going to need -moz-, so should list that one too.

So like this:

cursor: -moz-grab;
cursor: -webkit-grab;
cursor: grab;

Same for grabbing.

For grabbing, you can make a class for grabbing, then in the js you can just add and remove the grabbing class.

**Pull request closed without merging**